src/components/Reports/TeamTable.jsx

```diff
@@ -1,13 +1,82 @@
-import React from 'react';
+import { React, useState } from 'react';
 import { Link } from 'react-router-dom';
 import './TeamTable.css';
+import { Input, FormGroup, FormFeedback } from 'reactstrap';
 
 function TeamTable(props) {
   // Display project lists
   let TeamsList = [];
+  const canEditTeamCode = props.hasPermission('editTeamCode') || props.auth.user.role == 'Owner';
+
+  const EditTeamCode = ({team}) => {
+
+    const [teamCode, setTeamCode] = useState(team.teamCode);
+    const [hasError, setHasError] = useState(false);
+    const fullCodeRegex = /^[A-Z]-[A-Z]{3}$/;
+
+    const handleOnChange = (value, team) => {
+      props.updateTeam(team.teamName, team._id, team.isActive, value);
+    };
+  
+    const handleCodeChange = e => {
+      let value = e.target.value;
+      console.log(value);
```

Comment:
Advise removal of console.log

Reply:
Great catch! I missed this one when I deleted all of the console.log. I've removed this one and updated the PR, please check. Thank you!